src/uwuw/tests/uwuw_unit_tests_preprocessor.cpp

```diff
@@ -192,14 +192,14 @@ TEST_F(UWUWTest, materialMetadata) {
 
   // now read in the material library
   UWUW* uwuw = new UWUW(out_file);
-  std::map<std::string, pyne::Material> mat_lib = uwuw->material_library;
+  pyne::MaterialLibrary mat_lib = uwuw->material_library;
 
   // pull out the only material
-  pyne::Material mat = mat_lib["mat:CentreStack"];
+  pyne::Material mat = mat_lib.get_material("mat:CentreStack");
 
   EXPECT_EQ(mat.metadata["name"].asString(), "mat:CentreStack");
   EXPECT_EQ(mat.metadata["fluka_name"].asString(), "CENTREST");
-  EXPECT_EQ(mat.metadata["mat_number"].asInt(), 1);
+  EXPECT_EQ(mat.metadata["mat_number"].asInt(), 8);
```

Comment:
Do we know why this `mat_number` changed?  It probably doesn't matter, but it would be good to know why.

Reply:
it is because of the new way number get attributed when adding mat in the MaterialLibrary 
If a mat without number gets added, it gets: `mat.lib.size()+1`